.github/ISSUE_TEMPLATE/provider_key.yml

```diff
@@ -0,0 +1,24 @@
+name: Submit new Provider Key
+description: Submit a new OpenTofu Provider Key
+title: "Provider Key: "
+labels: ["provider-key", "submission"]
+body:
+  - type: input
+    id: namespace
+    attributes:
+      label: Provider Namespace
+      description: Path to a public GitHub repository following the pattern {owner}/terraform-provider-{name}, ex. opentofu/terraform-provider-aws
```

Comment:
I think this description is wrong, it should just be the namespace (organization/username)